File: 0001/solution.c
#include <stdio.h>

#define INPUT 1000

int solve(const int n) {
    int solution = 0;

    for(int i = 0; i < n; i++){
        if(i % 3 == 0 || i % 5 == 0){
            solution += i;
        }
    }

    return solution;
}

int main(void) {
    printf("%d\n", solve((const int) INPUT));
    return 0;
}

File: 0001/solution.py
#!/usr/bin/python3

def solve(n: int) -> (int):
    solution = 0
    for i in range(1, n):
        if i % 3 == 0 or i % 5 == 0:
            solution += i

    return solution

if __name__ == '__main__':
    print(solve(1000))

